docs/fides/docs/installation/docker.md

```diff
@@ -1,9 +1,36 @@
 # Installation from Docker
 
-For the ease of deployment in production, the community releases a production-ready reference container image.
+Docker is a platform as a service that bundles software into containers, each of which contains everything the software needs to run, software, libraries, and configuration files. [Docker Hub](https://docs.docker.com/docker-hub/) makes it easy to manage container images.
 
-The fidesctl community releases Docker Images which are reference images for fidesctl. Every time a new version of fidesctl is released, the images are available in the [ethyca/fidesctl DockerHub](https://hub.docker.com/r/ethyca/fidesctl/tags). There are also mid-release versions (dirty versions) that get uploaded to DockerHub on every commit to the main branch.
+The fidesctl community releases a production-ready reference container image of fidesctl for the ease of deployment in production.
 
-These reference images contain all of the extras and dependencies for running the Python application. However they do not contain the required Postgres database.
+## Docker Hub Releases
 
-Fidesctl requires multiple components to function as it is a multi-part application. You may therefore also be interested in launching fidesctl in the Docker Compose environment, see: [Running Fidesctl in Docker](../quickstart/docker.md).
+The following is an overview of the fidesctl Docker Hub release process, including release cadence and what's included. 
+
+**Release Cadences**
+
+Every time a new version of fidesctl is released, the images are available in the [ethyca/fidesctl DockerHub](https://hub.docker.com/r/ethyca/fidesctl/tags).
+
+There are also mid-release versions (dirty versions) that get uploaded to DockerHub on every commit to the main branch. Implementation is not finalized in mid-release versions. 
+
+**What's Included**
+
+These reference images contain all of the extras and dependencies for running the Python application. However they do not contain the required Postgres database. Continue to [Setting up the database](./database.md).
+
+## Installation
+
+Install Docker and Docker Compose, then pull the latest container image from Docker Hub.
+
+1. Install `docker` locally (see [Docker Desktop](https://www.docker.com/products/docker-desktop) or your preferred installation). The minimum verified Docker version is `20.10.8`
+2. If your `docker` installation did not include `docker-compose`, make sure to get at least version `1.29.0`. Installation instructions can be found in the [Docker Compose documentation](https://docs.docker.com/compose/install/).
```

Comment:
how would I know that my installation didn't include docker-compose? it may not be necessary to explicitly state instructions on how to check for it, but maybe link to instructions on how to confirm?